integration-test/Css.ts

```diff
@@ -251,6 +251,12 @@ class CssBuilder<T extends Properties1> {
   get bgBlue() { return this.add("backgroundColor", "#526675"); }
   get bgPrimary() { return this.add("backgroundColor", "var(--primary)"); }
   bgColor(value: string) { return this.add("backgroundColor", value); }
+  get fBlack() { return this.add("fill", "#353535"); }
```

Comment:
I went to look at what tailwinds calls their utility classes, and it looks like they have a single `fullCurrent` that then you just use with the existing color utilities:

https://tailwindcss.com/docs/fill

I don't know fill at all, but wdyt of their approach? I'm still good with this, just curious.

Reply:
Yea, seems interesting. We seem to be handling this by using `fill: inherit` on our icons in Beam.That way we can set the fill color on the parent element and have it passed down. `currentColor` is an interesting approach too. I'm guessing that coincides with using CSS variables to get it all to work.

Reply:
Actually just looked into currentColor some more... thats actually really cool! I didn't realize it was a built in css property. Definitely should use that rather than `inherit`, would probably make things a bit simpler. Thanks for sharing that!